### ISSUE TITLE
 get_tap_service  - DP02_01_Introduction

### ISSUE THREAD

**Issue by kcdage** [rubin-dp0/tutorial-notebooks#172]
Hello! 

In section 2.1 of the notebook, the code is 

> service=get_tap_service("tap")

This raises an error 
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Input In [3], in <cell line: 1>()
----> 1 service = get_tap_service("tap")

TypeError: get_tap_service() takes 0 positional arguments but 1 was given

We removed "tap" from the argument and it appears to be working, however.

**Comment by plazas** [comment]
Hi! Thank you for your report. Are you running this at `data.lsst.cloud`? if so, which version of the LSST Science Pipelines are you using? I just checked with version `w_2024_04`  ("recommended") and it seems to work.

**Comment by kcdage** [comment]
Yes! Two of us are using it through data.lsst.cloud. Can you let us know how to check which version of the pipeline we are running? 

thanks!

**Comment by plazas** [comment]
In the lower part of the notebook, you can see that information: 
<img width="466" alt="image" src="https://github.com/rubin-dp0/tutorial-notebooks/assets/4008497/08ea5a3f-c545-4c77-84ac-71812223a67c">

But you could also try to log out and then log back in again to the RSP, and then you will be prompted to choose an image for the Notebook aspect (you can choose "recommended" at that point). Let me know if it works!

**Comment by plazas** [comment]
@kcdage Did the notebook work for you when changing the pipeline version? Please let us know so we can assist you further or close this issue. Thank you!
